### PR TITLE
gpstart: verify heap_checksum matches on master and standby

### PR DESCRIPTION
**Pull request by nadeemg**
Signed-off-by: Nadeem Ghani <nghani@pivotal.io>

### CODE REVIEW

File: gpMgmt/bin/gpstart [hunk 1]
```diff
@@ -130,6 +130,12 @@ class GpStart:
                 # check that checksum settings are consistent across the cluster
                 num_workers = min(len(self.gparray.get_hostlist()), self.parallel)
                 heap_checksum = HeapChecksum(gparray=self.gparray, num_workers=num_workers, logger=logger)
+                if self.gparray.standbyMaster:
```
Comment (larham) [inline gpMgmt/bin/gpstart]: Accessing an attribute "standbyMaster" attribute seems like accessing a private.  Consider adding a method like "hasStandbyMaster()" to gparray.

File: gpMgmt/bin/gpstart [hunk 2]
```diff
@@ -130,6 +130,12 @@ class GpStart:
...
+                    if heap_checksum.get_master_value() != heap_checksum.get_standby_value():
+                        logger.warning("Heap checksum settings on standby master do not match master<<<<<<<<")
```
Comment (larham) [inline gpMgmt/bin/gpstart]: Does the requested format include an adjacent "<<<<"?  I was expecting a space after the wor "master" before "<<<<<", but if Scott wants it like that, fine with me.

Comment (skahler-pivotal) [reply gpMgmt/bin/gpstart]: In our other pieces that use that there is a space between the word and the <<<< so let's add a space

**Comment by Chibin** [comment]
@larham addressed your comments, please take another look

**Pull request merged**